essential-knowledge/index.rst

```diff
@@ -15,6 +15,7 @@ General
 
     infrastructure
     addons
+    ssl-certificates
```

Comment:
This is more important than addons, so put it one line up. After that, all good to merge.